doc/gettingstarted.md

```diff
@@ -112,7 +115,7 @@ This create an application `app1` for the user `admin`.
 You can now register an OTAA activated device to be used with the stack as follows:
 
 ```bash
-$ docker-compose exec stack ttn-lw-cli end-devices create --device-id dev1 --dev-eui 0004A30B001C0530 --join-eui 800000000000000C --application-id app1 --frequency_plan_id EU_863_870 --root_keys.app_key.key 752BAEC23EAE7964AF27C325F4C23C9A --lorawan_phy_version 1.0.2-b --lorawan_version 1.0.2
+$ ttn-lw-cli end-devices create --device-id dev1 --dev-eui 0004A30B001C0530 --join-eui 800000000000000C --application-id app1 --frequency_plan_id EU_863_870 --root_keys.app_key.key 752BAEC23EAE7964AF27C325F4C23C9A --lorawan_phy_version 1.0.2-b --lorawan_version 1.0.2
```

Comment:
```suggestion
$ ttn-lw-cli devices create app1 dev1 --dev-eui 0004A30B001C0530 --join-eui 800000000000000C --frequency_plan_id EU_863_870 --root_keys.app_key.key 752BAEC23EAE7964AF27C325F4C23C9A --lorawan_phy_version 1.0.2-b --lorawan_version 1.0.2
```

Reply:
`end-devices` was fine, this is the official term, and also how they're called in our API (except where #170 still needs to fix it). `device`, `dev`, `ed` and `d` are simply aliases for `end-devices`.

Reply:
OK for the official term. I'm aware of aliases.